CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix issue where VF stops working after changing device while muted
 - Fixed an issue for mute local when there is no audio input.
 - Fixed trucation of video subscriptions not occuring if the resubscribe was driven by `MonitorTask`.
```

Comment:
I would appreciate if you could update these two "Fixed" to "Fix".

Reply:
Fixed